stock_picking_package_info/models/stock.py

```diff
@@ -14,59 +14,36 @@ class StockPicking(models.Model):
     def _compute_num_packages(self):
         self.num_packages = sum(x.quantity for x in self.package_totals)
 
-    packages = fields.Many2many(
-        comodel_name='stock.quant.package',
-        relation='rel_picking_package', column1='picking_id',
-        column2='package_id', string='Packages', copy=False)
-    packages_info = fields.One2many(
-        "stock.picking.package.kg.lot", "picking", string="Packages Info",
-        readonly=True)
-    package_totals = fields.One2many(
-        "stock.picking.package.total", "picking",
-        string="Total UL Packages Info", readonly=True)
-    num_packages = fields.Integer(
-        string='Num. Packages', compute='_compute_num_packages', store=True)
-
     @api.multi
-    def action_assign(self):
-        super(StockPicking, self).action_assign()
-        self._delete_packages_information()
-        return True
+    @api.depends('pack_operation_ids', 'pack_operation_ids.result_package_id')
+    def _calc_picking_packages(self):
+        for picking in self:
+            picking.packages = [(6, 0, [])]
```

Comment:
Pedro esta instrucción ahora si hace falta, antes solo se ponía: 
        self.packages = [
            operation.result_package_id.id for operation in
            self.pack_operation_ids if operation.result_package_id]
Y packages quedaba con los paquetes de operaciones que encontraba, ahora no ahora si sel.packages tiene el id 3, y ahora solo hay una operación con paquete id 6, packages no se queda solo con el 6, sino que se queda con el 3 de antes + el 6 de ahora. Por eso he puesto está línea de código.